applications/lethe-fluid-matrix-free/mf_navier_stokes.cc

```diff
@@ -40,8 +40,17 @@ main(int argc, char *argv[])
           prm.parse_input(argv[1]);
           NSparam.parse(prm);
 
-          MFNavierStokesSolver<2> problem(NSparam);
-          problem.solve();
+          if (NSparam.stabilization.use_default_stabilization == true ||
+              NSparam.stabilization.stabilization ==
+                Parameters::Stabilization::NavierStokesStabilization::pspg_supg)
+            {
+              MFNavierStokesSolver<2, NavierStokesSUPGPSPGOperator<2, double>>
```

Comment:
Considering we will only have 2-3 alternative operator for a long time, this should not be that much of a big deal for the time being.